egs/wsj_mix/asr1/RESULTS.md

```diff
@@ -1,23 +1,75 @@
-# WSJ-2mix Result
-## word level rnnlm with / without speaker parallel attention
-## CER
+# Transformer Result
+## Pytorch backend Transformer without any hyperparameter tuning
+This code can be used for running experiments in [End-To-End Multi-Speaker Speech Recognition With Transformer](https://ieeexplore.ieee.org/abstract/document/9054029) paper. 
+ - Environments
+    - date: `Tue May  5 16:56:33 EDT 2020`
+    - python version: `3.7.3 (default, Mar 27 2019, 22:11:17)  [GCC 7.3.0]`
+    - espnet version: `espnet 0.6.0`
+    - chainer version: `chainer 6.0.0`
+    - pytorch version: `pytorch 1.0.1.post2`
+    - Git hash: `ae73f9b88cbec70bfa1325dd77042bbc289f419e`
+    - Commit date: `Sun May 3 20:15:00 2020 -0400`
+ - Model files (archived to model.tar.gz by `$ pack_model.sh`)
+    - training config file: `conf/train_multispkr_transformer.yaml`
+    - decoding config file: `conf/decode.yaml`
+    - cmvn file: `data/tr/cmvn.ark`
+    - e2e file: `exp/tr_pytorch_train_multispkr_transformer_no_preprocess_delta/results/model.last10.avg.best`
+    - e2e JSON file: `exp/tr_pytorch_train_multispkr_transformer_no_preprocess_delta/results/model.json`
+    - lm file: `exp/train_rnnlm_pytorch_lm_word65000/rnnlm.model.best`
+    - lm JSON file: `exp/train_rnnlm_pytorch_lm_word65000/model.json`
+ - Results
+    - WSJ\_2mix (model link: https://drive.google.com/open?id=10DH\_HhgFyzamq3QXqayxHJZMtxEf2oyR)
+
+ ```
+ exp/tr_pytorch_train_multispkr_transformer_no_preprocess_delta/decode_cv_decode_lm_word65000_model.last10.avg.best/min_perm_result.json
+ |  # Snt |   Corr     Sub     Del   Ins    Err   |
+ |  503   |   13996    1989    441   317    16.72 |
+ exp/tr_pytorch_train_multispkr_transformer_no_preprocess_delta/decode_tt_decode_lm_word65000_model.last10.avg.best/min_perm_result.wrd.json
+ |  # Snt |   Corr     Sub     Del   Ins    Err   |
+ |  333   |   10094    1055    169   200    12.58 |
+ ```
+
+    - WSJ0\_2mix (model link: https://drive.google.com/open?id=1xm2W1AXBgnccq-AFkp5-sDe2R0X8CMNx)
```

Comment:
I don't know why but this line becomes an inline code. Please fix it. 
(see https://github.com/espnet/espnet/blob/d24f502e942ebe0bea7e284a60fb01b8370a8d48/egs/wsj_mix/asr1/RESULTS.md)